Lib/test/test_future.py

```diff
@@ -75,6 +77,20 @@ def test_badfuture10(self):
             from test import badsyntax_future10
         self.check_syntax_error(cm.exception, "badsyntax_future10", 3)
 
+    def test_ensure_flags_dont_clash(self):
+        # bpo-39562: test that future flags and compiler flags doesn't clash
+
+        # obtain future flags (CO_***) from the __future__ module
```

Comment:
Do all these constants start with CO_FUTURE_? I'm asking for the comment.

Reply:
Yes, all future constants starts with `CO_FUTURE_`